### PR TITLE
delete claim attachment

### PR DESCRIPTION
**Pull request by Baggerone**
I also moved the previous function and test for attaching a claim file to the new files

### CODE REVIEW

File: application/actions/app.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ const (
 	batchesPath         = "/batches"
 	claimsPath          = "/" + domain.TypeClaim
 	claimItemsPath      = "/" + domain.TypeClaimItem
+	claimFilesPath      = "/" + domain.TypeClaimFile
```
Comment (jason-jackson) [inline application/actions/app.go:58]: Should be before claimItemsPath (if sorting alphabetically)

File: application/actions/claimfiles.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+//   '204':
+//     description: OK but no content in response
+func claimFilesDelete(c buffalo.Context) error {
+	tx := models.Tx(c)
+	cFile := getReferencedClaimFileFromCtx(c)
+	cFile.Destroy(tx)
+	return c.Render(http.StatusNoContent, nil)
+}
+
+// getReferencedClaimFileFromCtx pulls the models.Item resource from context that was put there
```
Comment (Schparky) [inline application/actions/claimfiles.go:75]:
```suggestion
// getReferencedClaimFileFromCtx pulls the models.ClaimFile resource from context that was put there
```

**Review by jason-jackson — APPROVED** [review]
Not all variables are alphabetized, but might be intentional.

File: application/models/claimfile.go [hunk 3]
```diff
@@ -54,6 +57,56 @@ func (c *ClaimFile) Create(tx *pop.Connection) error {
...
+	}
+
+	var claim Claim
+	if err := claim.FindByID(tx, c.ClaimID); err != nil {
+		panic(err.Error())
+	}
+
+	claim.LoadPolicy(tx, false)
+	policy := claim.Policy
+	policy.LoadMembers(tx, false)
```
Comment (Schparky) [inline application/models/claimfile.go:99]: I think `Policy.isMember` could be used here

**Review by Schparky — APPROVED** [review]

File: application/actions/claimfiles.go (hunk 2 above)
Comment (Baggerone) [reply application/actions/claimfiles.go:75]: That's funny> I corrected the old copy-paste error and left this one.

File: application/models/claimfile.go (hunk 3 above)
Comment (Baggerone) [reply application/models/claimfile.go:99]: :+1:

File: application/actions/app.go (hunk 1 above)
Comment (Baggerone) [reply application/actions/app.go:58]: :+1:

**Pull request merged**